src/Components/Color/Color.jsx

```diff
@@ -1,6 +1,20 @@
+import { useState } from "react";
 import "./Color.css";
 
-export default function Color({ color }) {
+export default function Color({ color, onDeleteColor }) {
+  const [showConfirmText, setShowConfirmText] = useState(false);
+
+  function handleDeleteColor() {
+    setShowConfirmText(true);
+  }
+
+  function handleConfirmDelete() {
+    onDeleteColor(color.id);
+  }
+
+  function handleCancelDelete() {
+    setShowConfirmText(false);
+  }
```

Comment:
very nice DIY solution, there is a standardised solution build into each browser called "confirm" but your own solution looks great too.

Reply:
Thanks, i will check the standard solution with "confirm" too. Thanks for the feedback:)